db/migrate/20160301140542_add_cloned_from_and_community_id_to_people.rb

```diff
@@ -0,0 +1,13 @@
+class AddClonedFromAndCommunityIdToPeople < ActiveRecord::Migration
+  def up
+    add_column :people, :cloned_from, :string, { limit: 22 }
+    add_index :people, :cloned_from
+    add_column :people, :community_id, :integer
```

Comment:
Currently our migrations run like 
duplicate people -> updated community_memberships table -> insert community_id value to people table
so we need to allow NULL values, or merge those migrations into one